src/lib/Behat/BusinessContext/NotificationContext.php

```diff
@@ -48,6 +48,15 @@ public function copiedToRootAppears(string $content): void
         $this->specificNotificationAppears($expectedMessage);
     }
 
+    /**
+     * @Then success notification that :locationName has been moved to trash appears
+     */
+    public function locationMovedToTrash(string $locationName): void
+    {
+        $expectedMessage = sprintf("Location '%s' moved to trash.", $locationName);
+        $this->specificNotificationAppears($expectedMessage);
+    }
+
```

Comment:
Why not just use the step `@Then success notification that :message appears`? Judging from the message format, readability would not suffer from that.

Reply:
I can change that, sure